### PR TITLE
Update README.md

### LINKED ISSUE #878: Bring README back up to par
Currently, the README is quite outdated and a bit messy. This should be amended with up-to-date launch instructions and more repo-oriented documentation. A nice addition would be some badges, like a go report card, and a link to the generated documentation on go.pkg.dev.

### PR DESCRIPTION
**Pull request by julesdesmit**
Fixes #878

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,54 +45,49 @@ Then, navigate to the testnet folder, like so:
...
 ```
 
-## Features
+## Wallet
 
-1. Cryptography Module - Includes an implementation of SHA-3 and LongsightL hash functions, Ristretto and BN-256 elliptic curves, Ed25519, BLS, bLSAG and MLSAG signature schemes, Bulletproofs zero-knowledge proof scheme.
-2. Consensus Module - Includes a complete implementation of the latest version of Segregated Byzantine Agreement \(v2.1\) consensus protocol, which contains three phases - Block Generation, Block Reduction and Block Agreement as well as the Blind Bid proof protocol utilized in the Block Generation phase.
-3. Networking and Database Module - Includes the blockchain storage and related logic, as well as the implementation of P2P gossip protocol utilized by the Dusk Network protocol.
-4. CLI Wallet - Includes an functionality enabling the user to create/load a wallet, transfer/bid/stake DUSK tokens, and managing the node. 
+The wallet is hosted in a separate folder, [found here](https://github.com/dusk-network/cmd/wallet).
```
Comment (autholykos) [inline README.md:62]: This link is broken. I think you forgot `dusk-blockchain` in the URL. However, currently there is no README in the `wallet/` folder

**Review by autholykos — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -57,54 +45,49 @@ Then, navigate to the testnet folder, like so:
...
+### How to use the wallet
 
 The wallet will show you a menu with available options, that you can navigate with the arrow keys and the enter key.
 
 Note that the wallet is a seperate process from the node, and thus closing the wallet does not stop the node from running.
 
-### How to claim Testnet DUSK?
+## License
+
+The Dusk Network blockchain client is licensed under the MIT License. See [the license file](LICENSE) for details.
```
[the remark this answers is not in the document]
Comment (julesdesmit) [reply README.md:89]: Not for this repo - all the Golang code is currently MIT licensed.

File: README.md (hunk 1 above)
Comment (julesdesmit) [reply README.md:62]: Good catch, will fix it

**Review by autholykos — APPROVED** [review]
LGTM

**Pull request merged**